tink-envelope-encryption-sample/README.md

```diff
@@ -0,0 +1,31 @@
+# Envelope encryption with Tink
+
+## Overview
+
+This guide provide an [envelope encryption](https://cloud.google.com/kms/docs/envelope-encryption) sample using [Tink](https://developers.google.com/tink) with Terraform.
+
+
+## Context
+
+This automation has 4 terraform modules: [0-bootstrap](./0-bootstrap/README.md), [1-encrypt](./1-encrypt/README.md), [2-decrypt](./2-decrypt/README.md) and [3-reencrypt-symetric-to-envelope](./3-reencrypt-symetric-to-envelope/README.md).
+
+**0-bootstrap** module is responsible to provide all the required infrastructure (locally and in GCP) in order to encrypt data using a tink encrypted [keyset](https://developers.google.com/tink/design/keysets).
+
+**1-encrypt** module is responsible to envelope encrypt a file using a tink encrypted [keyset](https://developers.google.com/tink/design/keysets).
+
+**2-decrypt** module is responsible to decrypt a file using a tink encrypted [keyset](https://developers.google.com/tink/design/keysets).
+
+**3-reencrypt-symetric-to-envelope** module is responsible to change a file encryption from [symetric](https://cloud.google.com/kms/docs/encrypt-decrypt) to [envelope](https://cloud.google.com/kms/docs/client-side-encryption#envelope_encryption_with_tink) using Tink.
```

Comment:
Can we split this third module into its own separate PR for easier review? Or is it tricky to untangle?

Reply:
Sure, it only requires a temporary change in `examples/ `because `1-encrypt `module is being tested implicitly by `3-reencrypt-symmetricto-envelope` module